app/views/pages/index.html.erb

```diff
@@ -1,9 +1,30 @@
-<div class="btn-group pull-right" data-toggle="buttons">
-    <% Filter.list.each do |filter| %>
-        <label id="<%= filter.filter_id %>" class="btn btn-primary" onclick="localStorage.setItem('jobfilter', '<%= filter.filter_id %>'); window.location = '?jobfilter=<%= filter.filter_id %>';" >
-          <input type="radio" name="<%= filter.filter_id %>" id="option_user" > <%= filter.title %>
-        </label>
-    <% end %>
+<div class="row pull-right">
```

Comment:
This is following the documentation of the grid system in Bootstrap:

![image](https://cloud.githubusercontent.com/assets/4260509/26074217/f6be6012-397e-11e7-9b11-a3d7569687f2.png)